packages/components/src/select-control/index.tsx

```diff
@@ -94,6 +95,12 @@ function UnforwardedSelectControl< V extends string >(
 
 	const classes = clsx( 'components-select-control', className );
 
+	maybeWarnDeprecated36pxSize( {
+		componentName: 'SelectControl',
+		__next40pxDefaultSize,
+		size: undefined,
```

Comment:
This component does have a `size` prop so it needs to be passed.

```suggestion
		size,
```